### ISSUE TITLE
Keybindings can shadow another mode

### ISSUE THREAD

**Issue by doolio** [NicolasPetton/pass#40]
Firstly, thank you for this package. Secondly, I can open this issue on your gitea instance if you prefer (and if possible). It's not clear if that's your preference from the README. 

If one has set `view-read-only` to a non-nil value then several bindings shadow view mode bindings. This has the effect of distorting the keybinding layout in the `*Password-Store*` buffer as the full M-x command is visible instead. The affected bindings are as follows:

w: pass-copy
r: pass-rename
u: pass-copy-url
n: pass-next-entry
p: pass-prev-entry
g: pass-update-buffer
o: pass-otp-options
RET: view entry (but the alternative v is displayed)

**Comment by doolio** [comment]
I've tried adding the following hook but to no avail.

```
(use-package pass
  :hook (pass-mode . (lambda ()
                       (setq-local view-read-only nil)))
  :commands pass)
```

**Comment by cvdub** [comment]
I fixed this in my pull request #46.

**Comment by NicolasPetton** [comment]
Fixed in PR #46.
